app/models/snap_application.rb

```diff
@@ -135,4 +160,36 @@ def signed_at_est
       in_time_zone("Eastern Time (US & Canada)")&.
       strftime("%m/%d/%Y at %I:%M%p %Z")
   end
+
+  private
+
+  def care_expenses_values
+    care_expenses.each do |value|
+      if !CARE_EXPENSES.include?(value)
+        errors.add(:care_expenses, "'#{value}' is an invalid care expense")
+      end
+    end
+  end
+
+  def medical_expenses_values
+    medical_expenses.each do |value|
+      if !MEDICAL_EXPENSES.include?(value)
+        errors.add(
+          :medical_expenses,
+          "'#{value}' is an invalid medical expense",
+        )
+      end
+    end
+  end
+
+  def court_ordered_expenses_values
+    court_ordered_expenses.each do |value|
+      if !COURT_ORDERED_EXPENSES.include?(value)
+        errors.add(
+          :court_ordered_expenses,
+          "'#{value}' is an invalid court-ordered expense",
+        )
+      end
+    end
+  end
```

Comment:
I'd post that Code Climate's complaint that these three methods are too similar, while it's valid, would become more complicated if you jump through the hoops to DRY it up. This is one of those cases where repetition is a by-product of clarity.

Thoughts, @jessieay?